app/indexers/file_set_indexer.rb

```diff
@@ -1,3 +1,8 @@
 class FileSetIndexer < Hyrax::FileSetIndexer
   self.thumbnail_path_service = IIIFWorkThumbnailPathService
+  def generate_solr_document
+    super.tap do |solr_doc|
+      solr_doc['hasRendering_ssim'] = object.rendering_ids
```

Comment:
Can we do this using a snake-case? That's the convention we're using almost everywhere else.